endorsement/dao/uwnetid_admin.py

```diff
@@ -0,0 +1,32 @@
+"""
+This module encapsulates the interactions with
+the UW NeTID Shared Admin resource
+"""
+
+import logging
+import traceback
+from uw_uwnetid.admin import get_admins_for_shared_netid
+from endorsement.dao import handel_err
+from restclients_core.exceptions import DataFailureException
+
+
+logger = logging.getLogger(__name__)
+
+
+def get_owner_for_shared_netid(netid):
+    """
+    Return admin resources
+    """
+    try:
+        for admin in get_admins_for_shared_netid(netid):
+            if admin.is_owner():
+                return admin.name
+    except DataFailureException as ex:
+        if ex.status == 404:
```

Comment:
What's this case? NetID change?

Reply:
shared netid ownership change

Reply:
Er, rather, I mean catching the 404 and passing

Reply:
more a deletion than change, but yeah